plugin/trino-influxdb/src/test/java/io/trino/plugin/influxdb/TestInfluxClient.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.trino.plugin.influxdb;
+
+import com.google.common.collect.ImmutableList;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.util.List;
+import java.util.Optional;
+
+import static io.trino.plugin.influxdb.InfluxConstant.ColumnKind.TAG;
+import static io.trino.plugin.influxdb.InfluxDataTool.TEST_DATABASE;
+import static io.trino.plugin.influxdb.InfluxDataTool.TEST_DATABASE_ANOTHER;
+import static io.trino.plugin.influxdb.InfluxDataTool.TEST_MEASUREMENT_X;
+import static io.trino.plugin.influxdb.InfluxDataTool.TEST_MEASUREMENT_Y;
+import static io.trino.plugin.influxdb.InfluxDataType.STRING;
+import static io.trino.plugin.influxdb.TestingInfluxServer.PASSWORD;
+import static io.trino.plugin.influxdb.TestingInfluxServer.USERNAME;
+import static org.assertj.core.util.IterableUtil.isNullOrEmpty;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertNotNull;
+import static org.testng.Assert.assertTrue;
+
+public class TestInfluxClient
```

Comment:
I couldn't find the benefit of this test class. Removing looks fine. Same for: 
* TestInfluxColumn
* TestInfluxColumnHandle
* TestInfluxSplit
* TestInfluxSplitManager
* TestInfluxTable
* TestInfluxTableHandle

Please note that we want to test the connector's behavior using query.